### ISSUE TITLE
MathJax equation: "Copy as MathML" no longer works

### ISSUE THREAD

**Issue by chchen2021** [typora/typora-issues#4635]
I used to be able to right click an equation, select "Copy Math | Copy as MathML" from the contextual menu, and directly paste into MS Word to create an equation. The copy function is now broken. "Copy Math | Copy as Tex" does not work either. The clipboard is empty in the Tex mode, and effectively empty in the MathML mode:

```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block"></math>
```

I am on Big Sur 11.5.2. The last time I used this copy-and-paste function is about a year ago, on Catalina. It was working perfectly back then.

**Comment by chchen2021** [comment]
Interestingly enough, "Copy Math | Copy as MathML" works for inline equations. It's just empty for display equations.

**Comment by abnerlee** [comment]
should be fixed in 1.1
